rule.go

```diff
@@ -370,9 +370,7 @@ func (r *Rule) Evaluate(tx *Transaction) []MatchData {
 			}
 
 		}
-		if r.Log {
-			tx.MatchRule(r, matchedValues)
-		}
+		tx.MatchRule(r, matchedValues)
```

Comment:
Can add checks here like, this fixes for #182 
```
if r.ID != 0 {
    tx.MatchRule(r, matchedValues)
}
```

Reply:
@ShiMing-Q is right, there is an issue with SecMarkers as they will be matched, we must add a validation like 
```go
if r.ID != 0 {
    tx.MatchRule(r, matchedValues)
}
```

Otherwise, SecMarkers will also be matched (SecMarkers have ID 0)